src/CommandResourceEstimator.cpp

```diff
@@ -0,0 +1,121 @@
+#include "CommandResourceEstimator.hpp"
+#include "CommandRunner.hpp"
+#include "Helpers.hpp"
+
+#include <google/protobuf/text_format.h>
+
+#include <process/defer.hpp>
+#include <process/dispatch.hpp>
+
+namespace criteo {
+namespace mesos {
+
+using namespace process;
+using namespace mesos;
```

Comment:
`using namespace` is pure evil. This is a bad practice and you don't need it since you have the other `using` below :)

Reply:
hum i guess i was confused with the way namespace should be use. It have been removed